opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/aggregation.py

```diff
@@ -131,13 +156,17 @@ def __init__(
         self._previous_collection_start_nano = self._start_time_unix_nano
         self._previous_value = 0
 
-    def aggregate(self, measurement: Measurement) -> None:
+    def aggregate(
+        self, measurement: Measurement, should_sample_exemplar: bool = True
+    ) -> None:
         with self._lock:
             if self._value is None:
                 self._value = 0
 
             self._value = self._value + measurement.value
 
+            super().aggregate(measurement, should_sample_exemplar)
```

Comment:
Does `offer` need to be within the `Aggregation` lock?

Reply:
I guess not